src/DataFixtures/ORM/LoadIdeaThreadData.php

```diff
@@ -124,6 +125,16 @@ public function load(ObjectManager $manager)
             false
         );
 
+        $threadDeletedOnPublishedIdea = Thread::create(
+            Uuid::fromString(self::THREAD_11_UUID),
+            '[Reduce Waste] Une discussion supprimée',
```

Comment:
This thread is concerned with the `Reduce Noise` idea and not  the `Reduce Waste`.